src/code/z_kanfont.c

```diff
@@ -1,7 +1,54 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_kanfont/func_8006EE60.s")
+// To be moved to message_data_static header
+#define MESSAGE_NEWLINE '\x01'
+#define MESSAGE_END '\x02'
+// message_data_static symbols
+extern const char D_07000000[]; // start of file
+extern const char D_070380D4[]; // charset message (id 0xFFFC)
+extern const char D_0703811C[]; // end marker of last message
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_kanfont/func_8006EEBC.s")
+void func_8006EE50(Font* font, s16 arg1, s16 arg2) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_kanfont/func_8006EF10.s")
+void Kanfont_LoadChar(u32 offset, u8 character, u16 codePointIndex) {
+    DmaMgr_SendRequest1(((u32)offset + codePointIndex) + 8,
+                        &_nes_font_staticSegmentRomStart[character * FONT_CHAR_TEX_SIZE], FONT_CHAR_TEX_SIZE,
+                        "../z_kanfont.c", 93);
+}
+
+void Kanfont_LoadMessageBoxEndIcon(Font* font, u16 icon) {
+    DmaMgr_SendRequest1(font->iconBuf, &_message_staticSegmentRomStart[4 * 0x1000 + icon * FONT_CHAR_TEX_SIZE],
+                        FONT_CHAR_TEX_SIZE, "../z_kanfont.c", 100);
+}
+
+void Kanfont_LoadOrderedFont(Font* font) {
```

Comment:
I think if we're taking out the "kan" in the struct/defines we might as well take it out from the system name as well
```suggestion
void Font_LoadOrderedFont(Font* font) {
```